lib/Elastica/Arrayable.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Elastica;
+
+/**
+ * Interface for params.
+ *
+ *
+ * @author Evgeniy Sokolov <ewgraf@gmail.com>
+ */
+interface Arrayable
```

Comment:
Ok, sorry, your arguments looks more reasonable :)

You right, it's subjective. I didn't seen much `*able` interfaces before and I'm just used to `Interface` suffix at the end of the name.